bundles/org.openhab.binding.freeboxos/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -235,6 +263,9 @@
 		<label>Ringing</label>
 		<description>Is the phone ringing</description>
 		<category>Alarm</category>
+		<tags>
+			<tag>Alarm</tag>
```

Comment:
Not sure it is appropriate.
Is it allowed now to set a Point tag without setting a Property tag?

Reply:
It is _**preferred**_ to have both a Point and a Property .. but not forbidden to have a Point alone. (By contrast it is forbidden to have a Property alone..)